admin/src/main/resources/io/buoyant/admin/js/src/routers.js

```diff
@@ -36,10 +35,6 @@ define([
    *   }
    */
   var Routers = (function() {
-
-    var clientRE = Query.clientQuery().withMetric("connects").build(),
-        serverRE = Query.serverQuery().withMetric("requests").build();
```

Comment:
I'd like to get rid of `routers.js` because I don't think it adds a lot of value on top of the fully hydrated metrics tree. I guess the adding clients logic is nice, but could just as easily be implemented in `metrics_collector.js`